src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -419,7 +419,7 @@ int map_device_using_suprocess(std::string arguments, int timeout_ms)
     case WAIT_OBJECT_0:
       if (!GetOverlappedResult(pipe_handle, &connect_o, &bytes_read, TRUE)) {
         err = GetLastError();
-        derr << "Couln't establish a connection with the child process. "
+        derr << "Couldn't establish a connection with the child process. "
```

Comment:
There was a request to expand the `mustn't` contraction. I didn't apply that throughout (I certainly can, but these PRs are already big enough, so I'd rather do it later...)